Source/Fuse.Reactive.Expressions/UnaryOperator.uno

```diff
@@ -34,6 +37,12 @@ namespace Fuse.Reactive
 		/** @deprecated Override the other `Compute` function. 2017-11-29 */
 		protected virtual object Compute(object operand) { return null; }
 
+		protected override sealed bool Compute(Argument[] args, out object result)
+		{
+			return Compute(args[0].Value, out result);
+		}
+		
+		/** @deprecated Override `Compute` or don't derive from `UnaryOperator` if you need argument tracking (which is rare). The typical base would be `Expression` and create a `Subscription` derived from `ExpressionListener`. 2017-12-14 */
 		protected virtual void OnNewOperand(IListener listener, object operand)
```

Comment:
The reason why you can't mark it as `[Obsolete("Override Compute instead")]` is that this would introduce warnings in the calling code, `InternalOnNewOperand`, right?

Reply:
By the way, I really wish we used `bool TryCompute(Argument[] args, out object result)` rather than `bool Compute(Argument[] args, out object result)`, so this wouldn't be so confusing. Is that too late to change? I don't think we've shipped this API-change yet...

Reply:
Yes, I have no strict path of obsolete entry points that I could mark the functions as obsolete. Non-obsolete ones need to call them for them to work. :/

As to `TryCompute`, yeah, it'd be better. There just used to be a `Compute` method in some of the operators. Let me verify we never shipped anything with `Compute` yet (since having `Compute` and `TryCompute` might be confusing)

Reply:
The `Unary/Binary/Ternary/Quaternary.Compute` virtuals have been there a long time. Some were marked `deprecated` before.  I'll see if there are enough new methods to justify calling it `TryCompute`.

Reply:
Or maybe not, the latest methods were introduced in https://github.com/fusetools/fuselibs-public/commit/ed7ed4f401dbb1ae0108dc15c94bb6705dd4ea6a  Has that been released?

Reply:
By the way, I guess we should consider adding support `#pragma warning disable XXX` etc to Uno so we could use obsolete code without having to rewire code like crazy until the full support can be removed...

Reply:
Yes, some pragma support would be nice. I'd love to see a language with actual features for library release cycles at some point.

BTW, I changed `Compute` to `TryCompute`